package.json

```diff
@@ -24,6 +24,7 @@
     "vue-moment": "^4.0.0",
     "vue-router": "^3.0.6",
     "vuetify": "^1.5.14",
+    "vuetify-confirm": "^2.0.0",
```

Comment:
@kvikshaug I think their abstraction of dialogs as a promise are worth it. We talked with Anna that we could copy a subset of their code and use it for input dialogs too

Reply:
Well, I'd be on board with the idea of forking the whole thing - we'll still get the abstraction and avoid the risks.

Reply:
Agreed with Luka to keep the package for now and replace it with our custom component later